velox/exec/Aggregate.cpp

```diff
@@ -57,21 +57,23 @@ AggregateRegistrationResult registerAggregateFunction(
     const std::string& name,
     const std::vector<std::shared_ptr<AggregateFunctionSignature>>& signatures,
     const AggregateFunctionFactory& factory,
+    const AggregateFunctionMetadata& metadata,
     bool registerCompanionFunctions,
     bool overwrite) {
   auto sanitizedName = sanitizeName(name);
   AggregateRegistrationResult registered;
 
   if (overwrite) {
     aggregateFunctions().withWLock([&](auto& aggregationFunctionMap) {
-      aggregationFunctionMap[sanitizedName] = {signatures, std::move(factory)};
+      aggregationFunctionMap[sanitizedName] = {
+          signatures, std::move(factory), std::move(metadata)};
```

Comment:
I dont think a std::move is required for the factory or metadata as they are const lvalue references  , adding a move will make a copy if I am not mistaken.

Reply:
Indeed. No point in moving const &. 

@shenh062326 You may want to change the signature to pass 'metadata' by value, then std::move would make sense.

Reply:
I have removed the std::move.